hostmap.go

```diff
@@ -22,7 +22,7 @@ const defaultPromoteEvery = 1000       // Count of packets sent before we try mo
 const defaultReQueryEvery = 5000       // Count of packets sent before re-querying a hostinfo to the lighthouse
 const defaultReQueryWait = time.Minute // Minimum amount of seconds to wait before re-querying a hostinfo the lighthouse. Evaluated every ReQueryEvery
 const MaxRemotes = 10
-const maxRecvError = 4
```

Comment:
@nbrownus Even though #1459 was a bug, I think correcting the logic but keeping the behavior is the way to go, thoughts?